### ISSUE TITLE
Limited list of supported devices

### ISSUE THREAD

**Issue by khaidakin** [Luligu/matterbridge-zigbee2mqtt#5]
I tried adding different zigbee devices to matterbridge-zigbee2mqtt. Of the 5 connected devices, matterbridge recognized 3. And these are the basic devices: a relay, a light bulb and a temperature/humidity sensor. Are there plans to support other types of devices in the future, including DIY devices that are not produced by companies?

![2024-03-29_11-59-32](https://github.com/Luligu/matterbridge-zigbee2mqtt/assets/59665419/86730d93-2c8c-40c4-b762-a7b43763f549)

![2024-03-29_12-07-04](https://github.com/Luligu/matterbridge-zigbee2mqtt/assets/59665419/afc16dfb-04b9-42d7-b6ae-2605968c138c)

**Comment by Luligu** [comment]
What are you adding? Maybe they are working can you send me please the 3 files bridge … info devices groups that you find in the .matterbridge directory?Actually can be easy to add everything but I need the 3 files… Do they work in z2m?On 29 Mar 2024, at 11:07, khaidakin ***@***.***> wrote:﻿
I tried adding different zigbee devices to matterbridge-zigbee2mqtt. Of the 5 connected devices, matterbridge recognized 3. And these are the basic devices: a relay, a light bulb and a temperature/humidity sensor. Are there plans to support other types of devices in the future, including DIY devices that are not produced by companies?
2024-03-29_11-59-32.jpg (view on web)
2024-03-29_12-07-04.jpg (view on web)

**Comment by khaidakin** [comment]
Yes, all devices work in zigbee2mqtt. I suggest starting with the simplest. with MAC address 0x00124b002a51fb6f. This is an 8 channel DIY relay. But instead of 8 channels, Home Assistant only has one channel.
[matterbridge.backup.json](https://github.com/Luligu/matterbridge-zigbee2mqtt/files/14802950/matterbridge.backup.json)
[matterbridge.json](https://github.com/Luligu/matterbridge-zigbee2mqtt/files/14802952/matterbridge.json)
[bridge-devices.json](https://github.com/Luligu/matterbridge-zigbee2mqtt/files/14802956/bridge-devices.json)
[bridge-info.json](https://github.com/Luligu/matterbridge-zigbee2mqtt/files/14802957/bridge-info.json)

**Comment by Luligu** [comment]
The first has multiendpoint and as specified in the known issues section they are still to implement (in my homebridge zigbee plugin I already have the code so it won't be too difficult).
The second is a thermostat that is still missed I will start with that one.
I will work on both issues.
Thanks

**Comment by Luligu** [comment]
Where did you buy the  0x00124b002a51fb6f. I have one similar but with 5 outlet. I'd love to buy 2 of them.... Do you have a link?

**Comment by khaidakin** [comment]
Thanks for the great product support!
It's a pleasure to work with you.
As for the 8 channel relays, I make them myself, like many other DIY devices. I have a link to these devices, but all the material is in Ukrainian, sorry. If you are interested, I will give you links to this

**Comment by Luligu** [comment]
That sound great can you send me picture of it?
I'm adding thermostat that was missed and then I will work about multi endpoint.

**Comment by Luligu** [comment]
Is a pleasure to work with you too.
Let's continue this way

**Comment by khaidakin** [comment]
There are 2 versions of the relay: with low-current contacts at 5A and high-current contacts up to 14A
![26588182](https://github.com/Luligu/matterbridge-zigbee2mqtt/assets/59665419/16e1187c-07be-4ea2-8d79-7eaeb02b9204)
![282797229](https://github.com/Luligu/matterbridge-zigbee2mqtt/assets/59665419/86fc2472-ca66-4e07-8b16-6d803924101e)
And there is a video of my blogger friend about this relay https://www.youtube.com/watch?v=ytuvblyAoo0

**Comment by Luligu** [comment]
They are very cool! Do you make them yourself?

**Comment by Luligu** [comment]
They will be Matter compliant in a few days...

**Comment by Luligu** [comment]
I need some help.
I made the cluster for Thermostat but I need to understand a few more stuff. Can you make a screenshot of expose page z2m for that device. I don't have it so ...

**Comment by khaidakin** [comment]
Sure!
![2024-03-29_22-48-51](https://github.com/Luligu/matterbridge-zigbee2mqtt/assets/59665419/4e89fd61-cbdf-4240-8836-bfb43470bea8)
![2024-03-29_22-49-23](https://github.com/Luligu/matterbridge-zigbee2mqtt/assets/59665419/f3866aac-615b-41b0-97b9-0394e4f4b8b1)
![2024-03-29_22-49-43](https://github.com/Luligu/matterbridge-zigbee2mqtt/assets/59665419/4ca755d2-fbd2-4a8d-8c09-e0ce55365309)

**Comment by Luligu** [comment]
Thanks so:
- we read the actual temperature from Local temperature
- we set the requested temperature with Current heating setpoint
- we turn the state off or heat with System mode
- we read the state from Running state
This is what is possible to do

**Comment by khaidakin** [comment]
OK. I understand the limitations and agree with you. 
But, if possible, then we need to add a Preset switch. 
This is important for thermostat control.

**Comment by Luligu** [comment]
Text me on Discord is better

**Comment by khaidakin** [comment]
I wrote to you on Discord
On which server should I look for you?

**Comment by khaidakin** [comment]
In three days, MatterBridge turned from a toy into a working tool. 
What have I tested in this and what works reliably without errors?
- single and multi-channel relays
- humidity-temperature sensors
- light bulbs
- thermostat
We already have real devices with which you can start building your smart home.
Many thanks to @Luligu for the great work done in a short time! 
I recommend this!
![2024-04-01_20-04-06](https://github.com/Luligu/matterbridge-zigbee2mqtt/assets/59665419/04816a24-0bfb-4c5d-b256-847e10e2ee50)
![2024-04-01_20-09-20](https://github.com/Luligu/matterbridge-zigbee2mqtt/assets/59665419/e43025dc-204d-4f6c-8c99-44c7fcdd5214)
![2024-04-01_20-09-51](https://github.com/Luligu/matterbridge-zigbee2mqtt/assets/59665419/4f38dfdb-8c7d-4d33-aad8-16603a3f3a54)
![2024-04-01_20-10-29](https://github.com/Luligu/matterbridge-zigbee2mqtt/assets/59665419/125a8cda-9c59-4c09-8258-31d2fadfa646)
![2024-04-01_20-10-56](https://github.com/Luligu/matterbridge-zigbee2mqtt/assets/59665419/6bcd19fe-75d0-4a99-bc67-c160c41df7d2)
![2024-04-01_20-11-17](https://github.com/Luligu/matterbridge-zigbee2mqtt/assets/59665419/c96a1f0c-6013-48ba-bc22-b22872c241dd)
![2024-04-01_19-26-31](https://github.com/Luligu/matterbridge-zigbee2mqtt/assets/59665419/ba78cf10-118c-4c2c-b819-72785cd1ba9d)
![2024-04-01_19-25-28](https://github.com/Luligu/matterbridge-zigbee2mqtt/assets/59665419/cd7e1fc5-86c7-4084-8de5-85d4d43a28c9)

**Comment by Luligu** [comment]
I'm flattered, thanks you so much for your help these last days.
